google_attachment/static/src/js/AttachmentBox.js

```diff
@@ -68,10 +99,12 @@ function addImportFromGoogleToSideBar(clientId, apiKey){
         _createAttachmentFromGoogleDriveDoc(doc){
             var attachmentValues = {
                 name: doc.name,
+                datas_fname: buildFileNameWithExtension(doc.name, doc.mimetype),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'datas_fname' is not in camel case. (camelcase)](https://app.codacy.com/app/numigi/odoo-web-addons/pullRequest?prid=3183161)